src/clndr.js

```diff
@@ -638,6 +638,15 @@
       eventType = 'touchstart';
     }
 
+    // Make sure we don't already have events
+    $container.off( eventType +'.clndr', '.'+this.options.targets.day )
+    .off( eventType +'.clndr', '.'+this.options.targets.empty )
+    .off( eventType +'.clndr', '.'+self.options.targets.previousButton )
+    .off( eventType +'.clndr', '.'+self.options.targets.nextButton )
+    .off( eventType +'.clndr', '.'+self.options.targets.todayButton )
+    .off( eventType +'.clndr', '.'+self.options.targets.nextYearButton )
+    .off( eventType +'.clndr', '.'+self.options.targets.previousYearButton );
+
```

Comment:
My mistake, will action these changes @mikegioia 

Shall I version bump and update the change log as well?